src/SearchParameters/index.js

```diff
@@ -344,9 +369,11 @@ SearchParameters.prototype = {
    * @return {SearchParameters}
    */
   setTypoTolerance : function setTypoTolerance( s ) {
```

Comment:
Can't we use a more explicit argument name, like in the in `setPage` or `setQuery`, like `typoTolerance` ?